test/conversations/Conversation.test.ts

```diff
@@ -22,14 +22,14 @@ describe('conversation', () => {
   let alice: Client
   let bob: Client
 
-  beforeEach(async () => {
```

Comment:
moving this down to the respective v1/v2 suites so that the client can be initialized accordingly.